src/main/java/codeu/controller/ServerStartupListener.java

```diff
@@ -30,6 +32,9 @@ public void contextInitialized(ServletContextEvent sce) {
 
       List<Message> messages = PersistentStorageAgent.getInstance().loadMessages();
       MessageStore.getInstance().setMessages(messages);
+      
+      List<Activity> activities = PersistentStorageAgent.getInstance().loadActivities();
```

Comment:
Does the ServerStartupListener basically just get all the data from datastore? So what is happening here is that once the server loads the activity feed data is loaded from datastore, correct?

Reply:
Yepp! That's exactly what's happening.